application/cms/forms.py

```diff
@@ -297,7 +297,7 @@ def __init__(self, is_minor_update: bool, sending_to_review=False, *args, **kwar
         self.lowest_level_of_geography_id.choices = choices
 
         if kwargs.get("obj", None):
-            self.internal_reference.data = kwargs["obj"].measure.reference
+            self.internal_reference.data = kwargs["obj"].measure.reference or ""
```

Comment:
I wonder if we need to write a test that covers this.

Reply:
:thinking: